src/index.html

```diff
@@ -4,13 +4,50 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, 
+      maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Search bars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-container">
+      <form 
+        class="search-box" 
+        data-qa="big"
+      >
+        <label for="search-input"></label>
+        <input 
+          type="text" 
+          id="search-input" 
+          class="search-input" 
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </form>
+    </div>
+
+    <div class="search-container">
+      <form 
+        class="search-box-small" 
+        data-qa="small"
+      >
+        <label for="search-input-small"></label>
+        <input 
+          type="text"
+          id="search-input-small"  
+          class="search-input-small, search-box input, search-box-small input" 
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+        
+        <label for="search-box"></label>
+        <label for="search-box-small"></label>
+        <label for="search-box, search-box-small"></label>
+        <input type="text" id="search-box" 
+        class= "search-box, search-box-small"
+        >
```

Comment:
please check the attributes
```suggestion
        <input type="text" id="search-box" class="search-box, search-box-small">
```